src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,33 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (this.getClass().equals(obj.getClass())) {
+            Rectangle rect = (Rectangle) obj;
+            if (width.equals(rect.width)
+                    && length.equals(((Rectangle) obj).length)
```

Comment:
do we need to cast obj again if we have already rect object?

Reply:
edited